release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Globalization.md

```diff
@@ -0,0 +1,22 @@
+# System.Globalization
+
+``` diff
+ namespace System.Globalization {
+     public class CultureNotFoundException : ArgumentException {
+-        protected CultureNotFoundException(SerializationInfo info, StreamingContext context);
```

Comment:
>protected CultureNotFoundException(SerializationInfo info, StreamingContext context); [](http://example.com/codeflow?start=9&length=85)

Is this how to show diff for methods which got a new attributes?

Reply:
Unfortunately yes, it's weird, I know.